doc/manuals/admin/build_source.md

```diff
@@ -42,6 +43,17 @@ commands that require root privilege):
         tar xfvz v1.0.2.tar.gz
         sudo mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include
 
+* Install libwebsocket from sources:
+
+        wget https://github.com/warmcat/libwebsockets/archive/v1.6.3.tar.gz
+        tar xfvz v1.6.3.tar.gz
+        cd libwebsockets-1.6.3/
+        mkdir build
+        cd build
+        cmake ..          # default cmake sets /usr/local as installation path
+        make
+        sudo make install
+
```

Comment:
Yeah, I think I also prefer a static lib ...
Easier for our users, as there will never be any problems with version mismatches.
It would be nice to see the difference is size though ... (after 'strip')